tests/test_public_cli.py

```diff
@@ -337,6 +337,22 @@ def test_fastani(
     )
 
 
+def test_sourmash(
+    tmp_path: str, input_genomes_tiny: Path, dir_sourmash_results: Path
+) -> None:
+    """Check sourmash run (default settings)."""
+    out = Path(tmp_path)
+    tmp_db = out / "example.sqlite"
+    public_cli.sourmash(
+        database=tmp_db, fasta=input_genomes_tiny, name="Test Run", create_db=True
+    )
```

Comment:
Here you should be overriding the defaults for scaled/num (which won't work for the viral genomes) with the values used to generate the fixtures.